src/docs/asciidoc/reference/manage/environment-access-control.adoc

```diff
@@ -1,3 +1,8 @@
 :relative-path: ../../
 include::{docdir}/variables.adoc[]
 
+.Environment support
+[NOTE]
+====
+The *environment* support is not completely done. It will be soon. The access control of the environments will be done via the *web console*. If you want to use environments you can anyway link:{contact-url}[contact us] to manage them for you.
```

Comment:
I think instead of "it will be soon", we should explain that it is our next high priority